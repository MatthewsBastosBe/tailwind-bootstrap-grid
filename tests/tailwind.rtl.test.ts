import { it } from 'vitest';

import run from './run-test';

// eslint-disable-next-line vitest/expect-expect
it('should handle rtl', () => run(__filename));
