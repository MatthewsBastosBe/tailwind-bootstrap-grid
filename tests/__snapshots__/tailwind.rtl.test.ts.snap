// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`should handle rtl 1`] = `
"/*! tailwindcss v4.1.5 | MIT License | https://tailwindcss.com */
@layer properties;
.fixed {
  position: fixed;
}
.order-1 {
  order: 1;
}
.order-5 {
  order: 5;
}
.order-first {
  order: -9999;
}
.order-last {
  order: 9999;
}
.col-4 {
  grid-column: 4;
}
.col-5 {
  grid-column: 5;
}
.col-6 {
  grid-column: 6;
}
.col-8 {
  grid-column: 8;
}
.col-12 {
  grid-column: 12;
}
.col-auto {
  grid-column: auto;
}
.container {
  width: 100%;
}
.flex {
  display: flex;
}
.grid {
  display: grid;
}
.border {
  border-style: var(--tw-border-style);
  border-width: 1px;
}
.capitalize {
  text-transform: capitalize;
}
@property --tw-border-style {
  syntax: "*";
  inherits: false;
  initial-value: solid;
}
@layer properties {
  @supports ((-webkit-hyphens: none) and (not (margin-trim: inline))) or
    ((-moz-orient: inline) and (not (color: rgb(from red r g b)))) {
    *,
    ::before,
    ::after,
    ::backdrop {
      --tw-border-style: solid;
    }
  }
}
"
`;
